### PR TITLE
migrate repsettings command

### PR DESCRIPTION
**Pull request by RainbowDashLabs**

### CODE REVIEW

File: src/main/java/de/chojo/repbot/commands/repsettings/handler/EmojiInfo.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+                emojiString(PROMPTED, "command.repsettings.emojidebug.message.prompted"),
+                emojiString(DONOR_LIMIT, "command.repsettings.emojidebug.message.donorLimit"),
+                emojiString(RECEIVER_LIMIT, "command.repsettings.emojidebug.message.receiverLimit")
+        );
+        return String.join("\n", emojis);
+    }
+
+    private String getBooleanMessage(EventContext context, boolean value, String whenTrue, String whenFalse) {
+        return context.localize(value ? whenTrue : whenFalse);
+    }
```
Comment (Taucher2003) [inline src/main/java/de/chojo/repbot/commands/repsettings/handler/EmojiInfo.java:55]: In the abuse protection command, this has been extracted to a utility interface. However, Abuse Protection migration is not yet merged so this is currently fine

File: src/main/resources/locale_es_ES.properties [hunk 2]
```diff
@@ -1,3 +1,4 @@
+>>>>>>>=d2aaf93 (migrate repsettings command)
```
Comment (Taucher2003) [inline src/main/resources/locale_es_ES.properties:1]: Conflict marker

File: src/main/resources/locale_pt_PT.properties [hunk 3]
```diff
@@ -1,3 +1,4 @@
+>>>>>>>=d2aaf93 (migrate repsettings command)
```
Comment (Taucher2003) [inline src/main/resources/locale_pt_PT.properties:1]: Conflict marker

**Review by Taucher2003 — CHANGES_REQUESTED** [review]

File: src/main/java/de/chojo/repbot/commands/repsettings/handler/EmojiInfo.java (hunk 1 above)
Comment (RainbowDashLabs) [reply src/main/java/de/chojo/repbot/commands/repsettings/handler/EmojiInfo.java:55]: Aye we will need to do some kind of general refactoring in the end. there is some stuff we can improve after merging all commands.

**Review by Taucher2003 — APPROVED** [review]

**Pull request merged**